test/nodejs/search.js

```diff
@@ -1240,7 +1240,8 @@ describe('Gix Position Tests', function () {
         "Kraft clinches ski jump title with final-event win",
         "Kraft wins final Planica event and ski-jumping World Cup",
         "Germany, Norway neck-and-neck after the first series in Planica",
-        "aa aa aa aa aa bb aa aa aa aa aa cc aa dd"
+        "aa aa aa aa aa bb aa aa aa aa aa cc aa dd",
+        "kk " + Array(1023).join("xx ") + "ll mm nn"
```

Comment:
Not instead, but in addition to current one with since 1023 is the modulo and it would be good to test around it a bit (e.g. 1022, 1024, 1025, 2047) to see how it works in the corner.